src/components/SearchInput/index.jsx

```diff
@@ -150,47 +154,72 @@ function SearchInput() {
 
     navigate(`/results?search_query=${keywords}`);
   }
-  function handleClick() {
+
+  function handleClickInput() {
```

Comment:
handleInputClick이 조금 더 괜찮은 함수 명 같습니다!
```suggestion
  function handleInputClick() {
```